### DISCUSSION TITLE
How to know which cells are used in `TilePlot`?

### DISCUSSION THREAD

**Discussion by YichaoOU** [stuart-lab/signac#1540] [Q&A]
Hello,

In addition to fragments of individual cells, I also want to know the gene expression. How to get the cell barcodes used by `TilePlot`?

Thanks,
Yichao

**Comment by timoast** [comment]
We don't return the cell barcodes used in the tile plot, but these are the cells with the highest counts for that region. You would need to quantify the counts in the plotted region yourself to determine which cells have the highest counts
